packages/contracts/src/RealityV3Oracle.sol

```diff
@@ -21,6 +19,10 @@ address constant REALITY_V3_ADDRESS = address(123456789); // will be replaced by
 /// with care to avoid unwanted results).
 /// @author Federico Luzzi - <federico.luzzi@protonmail.com>
 contract RealityV3Oracle is IOracle, Initializable {
+    address public immutable realitio;
```

Comment:
It's generally an alt name for Reality.eth, I'm assuming it comes from an old domain they were using. Anyway, I'll change that

Reply:
Fixed in 53c650ad3a0e129c0202cf0d4d7a948856465a56. The Reality.eth v3 arbitrator interface defines a function called `realitio` that returns the Reality.eth contract address, so that can't be changed.